docworks-dts/lib/multiple-files/generators.js

```diff
@@ -0,0 +1,212 @@
+const dom = require('dts-dom')
+const BASE_TYPES = {
+	string: dom.type.string,
+	boolean: dom.type.boolean,
+	number: dom.type.number,
+	Object: dom.type.any,
+	void: dom.type.void,
+	'*': dom.type.any
+}
+const trimParam = text => {
+	if (text) return text.trim().replace(/<(?:.|\n)*?>/gm, '')
+	return ''
+}
```

Comment:
should this be part of the `documentationGenerator`?

Reply:
Actually, I tried it and some tests were falling. in some cases, we just copy the docs summary without passing them through the `documentationGenerator`